<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDiskInfo</class>
 <widget class="QDialog" name="DialogDiskInfo">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>504</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>606</width>
    <height>504</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDiskInfo">
      <attribute name="title">
       <string>Disk info</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lIcon">
         <property name="text">
          <string>Icon:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelIcon">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lname">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditName">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lID">
         <property name="text">
          <string>ID:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelID">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelType">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSoftLocked">
           <property name="text">
            <string>Soft locked</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxGeosDisk">
           <property name="text">
            <string>Geos disk</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lDescription">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelDescription">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lDOS">
         <property name="text">
          <string>DOS:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="labelDOS">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lGeosID">
         <property name="text">
          <string>Geos ID:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="labelGeosID">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lGeometry">
         <property name="text">
          <string>Geometry:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="labelGeometry">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lBlocks">
         <property name="text">
          <string>Blocks:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="labelBlocks">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lFilename">
         <property name="text">
          <string>Filename:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="lineEditFilename">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lPath">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="lineEditPath">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBAM">
      <property name="toolTip">
       <string>Block Availability Map</string>
      </property>
      <property name="toolTipDuration">
       <number>5000</number>
      </property>
      <attribute name="title">
       <string>BAM</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutTabBAM">
       <item>
        <widget class="QTabWidget" name="tabBAMSides">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::East</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <widget class="QWidget" name="tabSideA">
          <attribute name="title">
           <string>Side A</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayoutSideA"/>
         </widget>
         <widget class="QWidget" name="tabSideB">
          <attribute name="title">
           <string>Side B</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayoutSideB"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHeader">
      <attribute name="title">
       <string>Header</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutTabHeader"/>
     </widget>
     <widget class="QWidget" name="tabDirectory">
      <attribute name="title">
       <string>Directory</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutTabDirectory">
       <item>
        <widget class="QComboBox" name="comboBoxDirectoryPages"/>
       </item>
       <item>
        <widget class="QLabel" name="labelDirectorySize">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDiskInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDiskInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
